include/RAJA/util/View.hpp

```diff
@@ -296,6 +297,76 @@ RAJA_INLINE AtomicViewWrapper<ViewType, AtomicPolicy> make_atomic_view(
   return RAJA::AtomicViewWrapper<ViewType, AtomicPolicy>(view);
 }
 
+struct layout_left{};
+struct layout_right{};
+
+template<typename LAYOUT>
+struct Reshape;
+
+template<typename LAYOUT>
+struct Reshape;
+
+template<std::size_t Id, std::size_t...Ids>
+constexpr auto get_first_index()
+{
+  return Id;
+}
+
+template<std::size_t...Is>
+struct Reshape<std::index_sequence<Is...>>
```

Comment:
@MrBurmark , this is the specialization that takes an index_sequence and identifies the correct unit stride

Reply:
Makes sense, I see an example is documentation coming?

Reply:
Yes, thanks for reminding me. I'll make something in the read the docs page within the next few days.